pkgs/development/python-modules/pwndbg/default.nix

```diff
@@ -19,6 +19,17 @@
   gdb-pt-dump,
   poetry-core,
 }:
+let
+  pwntools-beta = pwntools.overrideAttrs (oldAttrs: rec {
```

Comment:
could you add a comment above stating why this is needed (i.e. error or upstream issue link)?